integration-tests/src/test/java/org/zowe/apiml/functional/apicatalog/ApiCatalogAuthenticationTest.java

```diff
@@ -302,4 +304,18 @@ void givenOnlyValidCertificate_whenAccessNotCertificateAuthedRoute_thenReturnUna
                 .statusCode(HttpStatus.UNAUTHORIZED.value());
         }
     }
+
+    @Nested
+    @Tag("HealthEndpointProtectionDisabledTest")
+    class GivenHealthEndpointProtectionDisabled {
```

Comment:
it is enough to have 1 parametrized test for all components

Reply:
I counted Gateway, Discovery, and this one for ApiCatalog. What about tests in cloud-gateway and caching service?

Reply:
configuration is modified only for Catalog, Discovery and Gateway services.

Reply:
But the parameter is set on all of the services start.sh, so I'm assuming that on all of them it should work, otherwise why is it there?

Reply:
I think it needs to be verified if all these services are exposing the discoveryclient's registered services list, if they don't have configuration in java, most likely they don't. If this is the case, you can remove the parameter from start.sh since it will have no impact.

Reply:
yes, they don't have the configuration. I removed the protectedEndpoint property from start.sh for those services.